pkg/proc/variables.go

```diff
@@ -1207,6 +1207,11 @@ func extractVarInfoFromEntry(tgt *Target, bi *BinaryInfo, image *Image, regs op.
 			}
 		}
 		if cmem != nil {
+			paddingBytes := int(t.Common().ByteSize) - len(cmem.data)
```

Comment:
I think this should be moved inside newCompositeMemory in pkg/proc/mem.go (add an extra `size int` parameter to it).

Reply:
Thank you Alessandro! I have moved the padding logic to `newCompositeMemory`. Since the type of `CommonType.ByteSize` is `int64`, I add the extra parameter as `size int64`. Please take another look.